Documentation/capabilities-guide.md

```diff
@@ -300,6 +300,10 @@ without changing the executed images.
 The `--cap-retain` option to `rkt run` manipulates the `retain` capabilities set.
 The `--cap-remove` option manipulates the `remove` set.
 
+Capabilities specified from the command-line will replace all capability settings in the image manifest.
```

Comment:
So as soon as one of `--cap-{retain,remove}` is specified it will automatically disregard the set from the manifest? For my interest, are those also mutual exclusive? I think it should be exactly explained what has precedence or cancels other things out